MIGRATION.md

```diff
@@ -3,19 +3,37 @@
 API changes in MoveIt releases
 
 ## ROS Rolling
-- `lockSceneRead()` and `lockSceneWrite()` are now protected member functions, for internal use only. To lock the planning scene, use LockedPlanningSceneRO or LockedPlanningSceneRW:
+- [10/2023] Planning request adapters are now separated into PlanRequest (preprocessing) and PlanResponse (postprocessing) adapters. The adapters are configured with ROS parameter vectors (vector order corresponds to execution order). Please update your pipeline configurations for example like this:
+```diff
+- request_adapters: >-
+-     default_planner_request_adapters/AddTimeOptimalParameterization
+-     default_planner_request_adapters/ResolveConstraintFrames
+-     default_planner_request_adapters/FixWorkspaceBounds
+-     default_planner_request_adapters/FixStartStateBounds
+-     default_planner_request_adapters/FixStartStateCollision
+-     default_planner_request_adapters/FixStartStatePathConstraints
++ # The order of the elements in the adapter corresponds to the order they are processed by the motion planning pipeline.
++ request_adapters:
++   - default_planner_request_adapters/ResolveConstraintFrames
++   - default_planner_request_adapters/FixWorkspaceBounds
++   - default_planner_request_adapters/FixStartStateBounds
++   - default_planner_request_adapters/FixStartStateCollision
++ response_adapters:
++   - default_planner_response_adapters/AddTimeOptimalParameterization
```

Comment:
These should change to `default_plannING_request_adapters` and `default_plannING_response_adapters` as well.

Also I think `FixWorkspaceBounds` is now `ValidateWorkspaceBounds`.